GitUI/UserControls/RevisionGridClasses/DvcsGraph.cs

```diff
@@ -546,6 +557,7 @@ private void dataGrid_Scroll(object sender, ScrollEventArgs e)
             UpdateColumnWidth();
         }
 
+        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2002:DoNotLockOnObjectsWithWeakIdentity")]
```

Comment:
:bulb: I'm not convinced this is correct. I would prefer to see it moved to its own pull request and evaluated separately.